meilisearch/tests/index/test_index.py

```diff
@@ -43,21 +47,20 @@ def test_index_info_with_wrong_uid(self):
 
     def test_get_primary_key(self):
         """Tests an API call to get primary-key of an index in MeiliSearch"""
-        index = self.client.get_index(uid='movies_uid')
+        index = self.client.get_index(uid=self.index_uid)
         response = index.get_primary_key()
         assert response is None
 
     def test_update_index(self):
         """Tests an API call to update an index in MeiliSearch"""
-        index = self.client.get_index(uid='movies_uid')
+        index = self.client.get_index(uid=self.index_uid)
         response = index.update(primaryKey='objectID')
         assert isinstance(response, object)
         assert index.get_primary_key() == 'objectID'
 
```

Comment:
We could add a tests that tries to update an already existing primaryKey. Which I don;t think should work just to see if it raises correctly an error. 
This is more relevant once we have the error handler but I write it down here to get your opinion

Reply:
It is a good idea for a test 👍

Reply:
It seems like the SDK doesn't handle the exception, it just sets a primary_key if it was None befor, otherwise it deas nothing. This should be a separate Issue

Reply:
#84